### ISSUE TITLE
Voting: Users need more control over vote starting, pausing, re-starting, etc. 

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#860]
**Round 2 insights:** 

In 3 of the Round 2 tests, it became apparent that users need more control over being able to start and stop votes. 

- In one test, the user accidentally started the vote before it was the right time. She had published the topic into a group, and so it was instantly shared to them, and they instantly starting voting. She then closed the vote as she realised her mistake, but then wasn't able to open it again at the right time (when voting should actually have begun). She then realised she couldn't restart a vote in the same topic, and had to start again with a new topic, which in the real life situation (General Assembly elections) would have been a real mess. 

- In one test, the user had misunderstood the voting set-up and created a vote in a discussion topic that made no sense. After starting the vote, she realised she had misunderstood and it was wrong, so she closed it. She was then trying to see if she could edit and re-start the vote. She discovered she couldn't (via a lot of looking in the HC, took some time), so had to start a new topic for the voting part of her topic. Again in real life this would have really caused a mess. 

- In one test the user was creating the topic for his General Assembly with discussion and voting. When setting it up, he wanted to be able to set up the vote already, but for it not to actually start for 30 days, as the discussion would take place for 30 days before the vote. As it was we pretended 30 days had passed and he started the vote, but maybe that wouldn't have been a convenient day/time for him to log on and do so - he was keen to be able to get it sorted already (less to think about). 

**Suggestions:** 

@kevincrepin @ilmartyrk (Some may be more for a later phase I guess)
- Can we have a start date and time for votes, as well as a deadline?
- Can we have an option to pause a vote, and restart it later=? 
- Can we allow users to close, edit and re-start a vote? There would need to be some thought about this - maybe only if no-one has yet voted, or you can but you lose any votes cast in the first one. 
- As an interim until we have the latter (if we do), can we make it clearer to users that you can't edit a vote once you have started it, and you can't restart a vote once you have closed it?

**Comment by kevincrepin** [comment]
I would say definitely later phase because these are all new features compared to what's currently available.

**Comment by BeccaMelhuish** [comment]
@kevincrepin makes sense! Though could we still do this bit (to try to prevent problems arising from them not knowing that before they start or close a vote):

_As an interim until we have the latter (if we do), can we make it clearer to users that you can't edit a vote once you have started it, and you can't restart a vote once you have closed it?_

**Comment by BeccaMelhuish** [comment]
For example, we could add two lines here: 

X You can't edit a vote once you have started it
X You can only start one vote per topic

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/8824efe5-4424-44fe-990c-6a7feb4044de)

And we could add a line here. 

**Check your vote is exactly as you want it!** You can't edit or pause a vote once you've started it, and can only start one vote per topic.  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/a2a40b20-be04-4549-9270-8e93a45682e6)

@kevincrepin wdyt?

**Comment by kevincrepin** [comment]
To me this sounds good. I added it also in the designs.

**Comment by BeccaMelhuish** [comment]
Super, thanks!

@ilmartyrk I added pre-UX test label in case possible to have it before next admin (mobile) test - Thurs at noon :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish updated the dialog
